src/ngrest/render/RenderCrud.php

```diff
@@ -48,6 +75,22 @@ public function getView()
         return $this->_view;
     }
     
+    /**
+     * Set crud render view object
+     *
+     * @param \luya\admin\ngrest\render\RenderCrudView|string|array $view Object, classname string or Yii object config array
+     *
+     * @since 2.0.0
+     */
+    public function setView($value)
+    {
+        if (is_a($value, RenderCrudView::class)) {
+            $this->_view = $value;
+        } else {
+            $this->_view = Yii::createObject($value);
+        }
+    }
```

Comment:
regarding setView:

This looks good to me, i am just asking myself:

+ why would someone like to set a new view object (but this is okay to provided this option, just thinking loud)
+ I think the is_a check is not need, because unless an object is provided you can set whatever you want anyhow. Maybe we should use `Instance::ensure($value, 'yii\base\View'` but i am not sure this will work:
  + it should test whether the object is instance of yii\base\View
  + create the object if its an array configuration
  + leave the object like it is when an object is passed
  + If a string is provided it should take it from the components list (like `view` it will take Yii::$app->view).

Reply:
Thanks, `Instance::ensure` seems to work (i've added test for it).
For me the reason to override `RenderCrudView` was the need to run my angular service (polling) on CRUD interface, so i've made this addition in `registerAngularController`